redaxo/src/addons/phpmailer/lib/mailer.php

```diff
@@ -348,4 +357,158 @@ public static function errorMail(): void
             $addon->setConfig('last_log_file_send_time', time());
         }
     }
+
+
+    protected function microsoft365Send(): bool
+    {
+        $from = '' === $this->Sender ? $this->From : $this->Sender;
+        $to = array_map(function ($addr) {
+            return ['emailAddress' => ['address' => $addr[0], 'name' => $addr[1] ?? '']];
+        }, $this->getToAddresses());
+        $subject = $this->Subject;
+
+        // Korrektes Mapping: contentType klein schreiben!
+        // Body-Type für Graph-API anhand von AltBody bestimmen
+        if (!empty($this->AltBody)) {
+            $body = ['contentType' => 'html', 'content' => $this->Body];
+        } else {
+            $body = ['contentType' => 'text', 'content' => $this->Body];
+        }
```

Comment:
Passt das? Müssten nicht Body und AltBody ggf. beide gesendet werden, wenn beides gesetzt? So ist doch eigentlich das normale Verhalten.

Reply:
Korrekt, aber das geht bei der MS API nicht. Nur ein body.

Reply:
habs in die readme geschrieben.